velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -126,6 +135,10 @@ class SubstraitVeloxPlanConverter {
   void flattenConditions(
       const ::substrait::Expression& sFilter,
       std::vector<::substrait::Expression_ScalarFunction>& scalarFunctions);
+
+  std::unique_ptr<velox::memory::ScopedMemoryPool> scopedPool_ =
+      velox::memory::getDefaultScopedMemoryPool();
```

Comment:
Using default memory pool won't work in all cases. Can we modify the constructor to take pool as a parameter?

Reply:
change this.